app/react-native/src/preview/components/OnDeviceUI/navigation/index.tsx

```diff
@@ -1,8 +1,7 @@
 import React, { PureComponent } from 'react';
 import { View, SafeAreaView, StyleSheet } from 'react-native';
+// @ts-ignore TODO: add types for react-native-swipe-gestures
```

Comment:
or add a typings.d.ts with `declare module 'react-native-swipe-gestures'` if types are not available for this lib 😉

Reply:
added in 29ddf5aafc483b9571cac5d4378ff53d6c4ae185